manifests/p/picpick/picpick/6.0.0/picpick.picpick.installer.yaml

```
@@ -0,0 +1,39 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: picpick.picpick

```

I think we should move this package to align with `Publisher.Package`
```suggestion
PackageIdentifier: NGWIN.picpick
```